packages/generator/src/edmx-parser/v4/edm-types.ts

```diff
@@ -94,7 +94,7 @@ export interface EdmxActionImport extends EdmxNamed, EdmxNamespaced {
 /**
  * @internal
  */
-export interface EdmxFunction extends EdmxNamed, EdmxNamespaced {
```

Comment:
[q] I got confused. What is the different between the `EdmOperation` (was EdmxFunction` before and the `EdmxFunctionImport`? I guess there are two things in the Edmx?

Reply:
Perhaps it would make sense to create a `EdmxOperationImport`?

Reply:
[req] I think Florian also stumbled over this. So I guess in v4 you have the Function and FunctionImport and we filter out cases where there is no import to the function right? I think it would make sense to add some doc here to explain that.

Reply:
I added quite extensive documentation on how we now handle `EdmxOperationImport` and also added a note to the `EdmxOperation` that it is linked to `EdmxOperationImport`. Please let me know if this is good enough or if you would like it to be more detailed.